gradle-plugin/build.gradle

```diff
@@ -69,6 +69,15 @@ publishing {
     }
 }
 
+sourceSets.all {
```

Comment:
Is it possible to add a comment to explain what exactly this is doing?

Reply:
It doesn't seem to fix the issue. I found this here - https://github.com/google/guava/issues/6617#issuecomment-1615502877

This PR is still in draft because I haven't yet worked out to fix the gradle test issues that I'm getting.

Reply:
removed - I upgraded Gradle in #175 and that sorted out the issue